docs/jobs/openings/web3.md

```diff
@@ -0,0 +1,34 @@
+## Web3 Developer Opening at Aragon
+
+Aragon is an open source, decentralized project fighting for the freedom to organize. We believe that decentralized organizations can solve the world’s worst problems.
+
+We’re looking for a Web3 Developer to help us consolidate smart contracts with a frontend, we call this position The Glue. Our Web 3 Developer must not only be a Javascript rockstar but also a decentralization evangelist, someone who enjoys researching and testing new Web3 technologies.
+
+### Responsibilities
+
+- Take ownership of the development and maintenance of [aragon.js](https://github.com/aragon/aragon.js), the JavaScript library between smart contracts and the native web experience.
+- Research and experiment with new Web3 technologies, including but not limited to [Web3.js](https://github.com/ethereum/web3.js/) and [Parity systems](https://github.com/paritytech).
+- Assist on smart contract and frontend development when needed.
+- Help create developer tools and technical documentation.
+
+### Requirements
+
+- Track record working with Javascript environments.
+- Expert understanding on ES6/ES7, React, client-side storage, resource optimization, CSS, using processors such as LESS or SASS, RxJS, service workers, and device APIs.
```

Comment:
I'd remove the `React` and `CSS, using processors such as LESS or SASS`, and `device APIs` bits, to replace them with `functional programming` and `Redux / Elm-inspired architectures`.

Reply:
The Web3 developer is probably not going to spend a lot of time using CSS and we haven't used a single device API, so I agree with Brett.

Reply:
That description is very similar to the frontend position listing, on that, the description was changed to

> Modern JavaScript (ES6+) and Web APIs, CSS, React (Web and Native), resource optimization, development tools.

Like the new suggestions :+1:

Reply:
I was specially confused on this end, so thanks for the input!